exporter/splunkhecexporter/logdata_to_splunk.go

```diff
@@ -58,23 +65,22 @@ func mapLogRecordToSplunkEvent(res pdata.Resource, lr pdata.LogRecord, config *C
 		fields[traceIDFieldKey] = traceID
 	}
 	if lr.SeverityText() != "" {
-		fields[splunk.SeverityTextLabel] = lr.SeverityText()
+		fields[severityTextKey] = lr.SeverityText()
 	}
 	if lr.SeverityNumber() != pdata.SeverityNumberUNDEFINED {
-		fields[splunk.SeverityNumberLabel] = lr.SeverityNumber()
+		fields[severityNumberKey] = lr.SeverityNumber()
 	}
 
 	res.Attributes().Range(func(k string, v pdata.AttributeValue) bool {
 		switch k {
-		case conventions.AttributeHostName:
+		case hostKey:
 			host = v.StringVal()
-			fields[k] = v.StringVal()
-		case splunk.DefaultSourceLabel:
+			fields[conventions.AttributeHostName] = v.StringVal()
+		case sourceKey:
 			source = v.StringVal()
-			fields[k] = v.StringVal()
```

Comment:
This line removal seems to be a functionality change, unrelated to the goal of this PR. I support it, but we should make it explicit and submit a separate PR. I can handle that later, so we can just keep it for now making sure that this PR is no-op when exporter used with default configuration.

Reply:
done, will deal with this separately indeed.